app/globals.css

```diff
@@ -7,3 +7,12 @@
   background-size: cover;
   border-radius: 0;
 }
+
+.mapboxgl-ctrl-group button {
+  width: 40px;
+  height: 40px;
+}
+
+.mapboxgl-ctrl-bottom-right {
+  margin-right: 16px;
```

Comment:
I checked the DOM and it looks like you have to do this bc of the attribution/copyright in the lower-right corner?